src/Redux/Crypto.js

```diff
@@ -0,0 +1,27 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState = [];
+const URL = 'https://api.coincap.io/v2/assets';
+
+export const getCryptos = createAsyncThunk('getting Api from cryptos', async () => {
```

Comment:
[OPTIONAL]

- I think it will be better if you declare your action types as a constant instead of raw string because raw strings can be prone to error which can result in a bug that might not be noticeable in your app as your app grows bigger. 

What if you want to change the name of the action type? It's good to just go change the value in the declared constant, rather than do a big find and replace across your whole project, hoping you don't overwrite something you didn't mean to in the process.

Plus it's just nice to have all your action types in one place, so you can go through and see what you have available, rather than looking through all your action files. Once your project grows, you'll be glad you did it.

e.g `const GET_CRYOTOS = 'getting Api from cryptos'`